src/Command/UpdateLicensesCommand.php

```diff
@@ -130,8 +130,8 @@ protected function configure()
                 'exclude',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Comma-separated list of folders and files to exclude from the update',
-                implode(',', static::DEFAULT_FILTERS)
+                'Comma-separated list of folders to exclude from the update',
+                implode(',', self::DEFAULT_FILTERS)
```

Comment:
Looks like this line is a partial revert of https://github.com/PrestaShopCorp/header-stamp/pull/27, and having the `static::` was a good thing.

Is there any reason for this change?

Reply:
I can't remember. This could be skipped.

Reply:
oops, merged bad branch.
I close this and reopen with a clean PR.